opcua/adapter.go

```diff
@@ -46,11 +46,11 @@ type Service interface {
 type Config struct {
 	ServerURI string
```

Comment:
Is there a reason this doesn't have a struct tags?

Reply:
The `ServerURI` and `NodeID` are created by the dynamically by the service and it is not initialized on service start.